product_equivalent_category/models/product_equivalent_category.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductEquivalentCategory(models.Model):
+    _name = "product.equivalent.category"
+    _description = "Product Equivalent Category"
+
+    name = fields.Char("Name", index=True, required=True, translate=True)
+    product_count = fields.Integer(
+        "# Products",
+        compute="_compute_product_count",
+        help="The number of products under this category",
+    )
+
+    @api.multi
```

Comment:
api.multi is deprecated

Reply:
Is deprecated in v13 not in v12

Reply:
I prefer to use read_group for a simple count
<pre>
categ_data = self.env["product.template"].read_group([
    ('equivalent_categ_id', 'in', self.ids),
], ['equivalent_categ_id'], ['equivalent_categ_id'])
mapped_data  = {
    record['equivalent_categ_id'][0]: record['equivalent_categ_id_count']
    for record in data
}
for categ in self:
    categ.product_count = mapped_data.get(categ.id, 0)
</pre>

Reply:
I agree to use read_group https://github.com/OCA/product-attribute/pull/574#discussion_r382529856

Reply:
@AdriaGForgeFlow

Reply:
Done :+1: